data/src/main/java/com/teamagam/gimelgimel/data/config/Constants.java

```diff
@@ -3,7 +3,7 @@
 import android.graphics.Bitmap;
 
 public class Constants {
-    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev1.herokuapp.com";
+    public static final String MESSAGING_SERVER_URL = "http://ggmessaging-dev2.herokuapp.com";
```

Comment:
I wanted it to change.
older versions still running would work vs dev1, where sensor messages are not supported, while current versions will work with dev2 that contains sensor messages.